overlay.nix

```diff
@@ -101,19 +86,19 @@ in rec {
 
   solc-versions =
     let
-      fetchNixpkgs = { owner, attr }:
+      fetchSolcVersions = { owner, attr }:
```

Comment:
These changes I've just ninja'd in, they have no relationship to this PR :)

Reply:
is it just name changes or does this change the semantics?

Reply:
Only name